src/deprecated/deprecated.js

```diff
@@ -1154,6 +1157,113 @@ Application.prototype.enableVr = function () {
     Debug.deprecated('pc.Application#enableVR is deprecated, and WebVR API is no longer supported.');
 };
 
+Application.prototype.setAreaLightLuts = function(asset) {
+    if (asset) {
+        const device = this.graphicsDevice;
+        asset.ready((asset) => {
+            AreaLightLuts.set(device, asset.resource);
+        });
+        this.assets.load(asset);
+    } else {
+        Debug.warn("setAreaLightLuts: asset is not valid");
+    }
+};
+
+AreaLightLuts.prototype.set = function (device, resource) {
```

Comment:
that's a large function being deprecated. And the refactored function uses majority of the code. I'd suggest to change this deprecated function to call the new function?